container/README.md

```diff
@@ -13,40 +13,30 @@ npm install @luigi-project/container
 
 2. Import it into your project:
 
-
 ```
-import '@luigi-project/container';
+import './bundle.js';
```

Comment:
```suggestion
import '@luigi-project/container';
```
I would say revert this, since bundle.js is too specific, `@luigi-project/container` should be a standard way to refer to the import